Tests/DatadogTests/Datadog/RUM/RUMMonitor/Scopes/RUMViewIdentityTests.swift

```diff
@@ -39,8 +41,10 @@ class RUMViewIdentityTests: XCTestCase {
         // Then
         XCTAssertTrue(identity1.equals(key1))
         XCTAssertTrue(identity2.equals(key2))
+        XCTAssertTrue(identity1.equals(identity1))
         XCTAssertFalse(identity1.equals(key2))
         XCTAssertFalse(identity2.equals(key1))
+        XCTAssertFalse(identity1.equals(identity2))
     }
```

Comment:
As now `equals()` accepts `nil`, I think we need to cover more cases in this file.